File: 00-每日一题/20200215_27.cpp
//
// Created by zanbo on 2020/2/15.
//
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int res=0;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]!=val)
                nums[res++] = nums[i];
        }
        return res;
    }
};

File: 00-每日一题/20200215_27.py
# 移除元素
class Solution:
    def removeElement(self, nums, val):

        j = 0

        for num in nums:
            if num == val:
                continue
            else:
                nums[j] = num
                j += 1
        return j


if __name__ == '__main__':
    obj = Solution()
    while True:
        nums_str = input().strip().split()
        nums = list(map(int, nums_str))
        val = int(input().strip())

        res = obj.removeElement(nums, val)
        print(res)